input.go

```diff
@@ -49,6 +71,38 @@ func MouseClickXY(x, y float64, opts ...MouseOption) MouseAction {
 	})
 }
 
+
+
+
+
+func MouseMoveNode(n *cdp.Node, opts ...MouseOption) MouseAction {
+	return ActionFunc(func(ctx context.Context) error {
```

Comment:
`MouseClickNode` do some check and scroll the node into view, should we do the same here? If the answer is yes, maybe we should  extract those code into a func.

Reply:
The MouseMoveNode function is the similar function to the MouseClickNode, but it doesn’t do the click movement, just put the mouse on the target node.